src/index.ts

```diff
@@ -1125,6 +1118,25 @@ class Datastore extends DatastoreRequest {
         // This was replaced with a more efficient mechanism in the top-level
         // `excludeFromIndexes` option.
```

Comment:
is this TODO still relevant? Does the top-level excludeFromIndexes work the same way as these changes?

Reply:
I think this TODO can be removed. There is currently code [here](https://github.com/googleapis/nodejs-datastore/blob/bbcfc3e73c56e3aa8152ecc9395d3c94133ea6b4/src/index.ts#L1138-L1150) that adds excludeFromIndexes in the top level and I added code that provides `excludeFromIndexes` everywhere else based on the path that gets generated. To avoid test failures, the code for adding `excludeFromIndexes` in the top level is still needed so I don't think this TODO is relevant anymore.